web/templates/settings/organization_permissions_admin.hbs

```diff
@@ -341,6 +341,15 @@
                 {{> settings_save_discard_widget section_name="other-permissions" }}
             </div>
             <div class="m-10 inline-block organization-permissions-parent">
+                {{#unless server_can_summarize_topics}}
+                    <div class="tip">
+                        {{t "AI summaries are not enabled on this server."}}
+                    </div>
+                {{/unless}}
+                {{> group_setting_value_pill_input
```

Comment:
Do we want to hide these settings in production for now?

Reply:
I don't think so, but we will want a custom strong for Cloud and/or some Help Center documentation around it.

Reply:
Yeah, so for Cloud, let's use:

> AI summaries are not available on Zulip Cloud yet.